### ISSUE TITLE
VSCode version 1.22 seems to have broken Debugging

### ISSUE THREAD

**Issue by PavloTitenok** [felixfbecker/vscode-php-debug#265]
Same issue as the one registered for 1.12.2
https://github.com/felixfbecker/vscode-php-debug/issues/203

Debug worked OK before update but now it always prompts: "Debug adapter process has terminated unexpectedly"
Basic version of launch.json for PHP is used:

```
 {
     "version": "0.2.0",
     "configurations": [
         {
             "name": "Listen for XDebug",
             "type": "php",
             "request": "launch",
             "port": 9000,
         },
         {
             "name": "Launch currently open script",
             "type": "php",
             "request": "launch",
             "program": "${file}",
             "cwd": "${fileDirname}",
             "port": 9000
         }
     ]
}
```

**Comment by niccolomeloni** [comment]
Same problem here with 1.22.2.

**Comment by balint777** [comment]
+1

**Comment by felixfbecker** [comment]
Please don't +1 issues. It clutters the thread without adding value to the discussion and spams maintainers with notifications. Use GitHub reactions to upvote features.
![image](https://cloud.githubusercontent.com/assets/10532611/18344735/bb60c2f6-75b8-11e6-88a9-822fae298110.png)

**Comment by PavloTitenok** [comment]
It looks like the issue was resolved in 1.23 version. I had to remove the extension and reopen vscode (reload did not help) and now debug works with exactly the same configuration.
